tests/networkpolicy_test.go

```diff
@@ -1,173 +1,234 @@
-			By("Connect vmia from vmib")
-			assertPingFail(vmib, vmia)
-		})
+			BeforeEach(func() {
+				// deny-by-default networkpolicy will deny all the traffice to the vms in the namespace
+				policy = createNetworkPolicy(serverVMI.Namespace, "deny-by-default", metav1.LabelSelector{}, []networkv1.NetworkPolicyIngressRule{})
+			})
+
+			AfterEach(func() {
+				waitForNetworkPolicyDeletion(policy)
+			})
+
+			It("[test_id:1511] should fail to reach serverVMI from clientVMI", func() {
+				By("Connect serverVMI from clientVMI")
+				assertPingFail(clientVMI, serverVMI)
+			})
+
+			It("[test_id:1512] should fail to reach clientVMI from serverVMI", func() {
+				By("Connect clientVMI from serverVMI")
+				assertPingFail(serverVMI, clientVMI)
+			})
+			It("[test_id:369] should deny http traffic for ports 80/81 from clientVMI to serverVMI", func() {
+				assertHTTPPingFailed(clientVMI, serverVMI, 80)
+				assertHTTPPingFailed(clientVMI, serverVMI, 81)
+			})
 
-		It("[test_id:1512] should be failed to reach vmib from vmia", func() {
-			By("Connect vmib from vmia")
-			assertPingFail(vmia, vmib)
 		})
 
-	})
+		Context("and vms limited by allow same namespace networkpolicy", func() {
+			var policy *networkv1.NetworkPolicy
 
-	Context("vms limited by allow same namespace networkpolicy", func() {
-		var policy *v1network.NetworkPolicy
-
-		BeforeEach(func() {
-			// allow-same-namespave networkpolicy will only allow the traffice inside the namespace
-			By("Create allow-same-namespace networkpolicy")
-			networkpolicy := &v1network.NetworkPolicy{
-				ObjectMeta: v13.ObjectMeta{
-					Name: "allow-same-namespace",
-				},
-				Spec: v1network.NetworkPolicySpec{
-					PodSelector: v13.LabelSelector{},
-					Ingress: []v1network.NetworkPolicyIngressRule{
+			BeforeEach(func() {
+				// allow-same-namespace networkpolicy will only allow the traffice inside the namespace
+				By("Create allow-same-namespace networkpolicy")
+				policy = createNetworkPolicy(serverVMI.Namespace, "allow-same-namespace", metav1.LabelSelector{},
+					[]networkv1.NetworkPolicyIngressRule{
 						{
-							From: []v1network.NetworkPolicyPeer{
+							From: []networkv1.NetworkPolicyPeer{
 								{
-									PodSelector: &v13.LabelSelector{},
+									PodSelector: &metav1.LabelSelector{},
 								},
 							},
 						},
 					},
-				},
-			}
-			var err error
-			policy, err = virtClient.NetworkingV1().NetworkPolicies(vmia.Namespace).Create(networkpolicy)
-			Expect(err).ToNot(HaveOccurred())
-		})
+				)
 
-		AfterEach(func() {
-			waitForNetworkPolicyDeletion(policy)
-		})
+			})
 
-		It("[test_id:1513] should be successful to reach vmia from vmib", func() {
-			By("Connect vmia from vmib in same namespace")
-			assertPingSucceed(vmib, vmia)
-		})
+			AfterEach(func() {
+				waitForNetworkPolicyDeletion(policy)
+			})
+
+			It("[test_id:1513] should be successful to reach serverVMI from clientVMI", func() {
```

Comment:
typo:   should be successful to <-> should succeed to

Reply:
Done